src/mock_adapter/controllers/user.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import appier
+import appier_extras
+
+import models
+
+class UserController(appier.Controller):
+
+    @appier.route("/login/", "GET")
+    def login(self):
+        #self.url_for("admin.login_api", username = "root", password = "root")

```

Comment:
don't leave commented code, if the user controller is no longer being used then delete the controller (and the model as well)